src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+
+  --text-color:#000;
+  --hover-color:#00ACDC;
+  ;
+}
+
+body, div, p, h1, h2, h3,a,li,ul,nav {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100vw;
+}
+
+.container {
+  margin: 0 50px;
+  display: flex;
+  padding: 10px 0;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+}
+
+.navorder {
+  display: flex;
+  list-style: none;
+}
+
+.navelement > a {
+  text-decoration: none;
+  color: var(--text-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;;
+  text-align: left;
+  text-transform: uppercase;
+
+}
+
+.navelement>a:hover {
+  color:var(--hover-color);
+}
+
+.navelement:not(:first-child){
+  margin-left: 20px;
+}
+
+.is-active{
+  position: relative;
+  padding: 0;
+}
+
+.is-active::after {
+  content: "";
+  display: block;
+  height: 4px;
+  background-color: var(--hover-color);
+  width: 0;
```

Comment:
the pseudo element should always display, only a text color changes when hovering
and the view is wrong, because pseudo-element without fillets and sticks to text
![image](https://github.com/mate-academy/layout_moyo-header/assets/43861663/d4b064c8-b547-4702-bcce-be3ec93ec42e)